app/models/candidate_interface/degrees_form.rb

```diff
@@ -16,9 +23,26 @@ def save_base(application_form)
         qualification_type: qualification_type,
         subject: subject,
         institution_name: institution_name,
+        grade: grade,
+        award_year: award_year,
       )
 
       true
     end
+
+  private
+
+    def other_grade?
```

Comment:
nitpick: is not rails convention to start this kind of method with 'is_' ?
`is_other_grade`, `is_predicted_grade?`